.github/workflows/fleetctl-profiles.yml

```diff
@@ -38,21 +38,16 @@ jobs:
       matrix:
         go-version: ['^1.19.4']
     steps:
```

Comment:
we don't need to define this matrix anymore, right?

Reply:
Good point. Updated!